### DISCUSSION TITLE
article/lagrange-polynomial/

### DISCUSSION THREAD

**Discussion by giscus** [3-24/3-24.github.io#22] [Comments]
# mathematics/numerical%20analysis/2/

함수의 정확한 형태가 주어지지 않고, 몇몇 점들의 함수값들만 알고 있다면 미분을 어떻게 근사할 수 있을까?

https://blog.youngseok.dev/mathematics/numerical%20analysis/2/

**Comment by pisache** [comment]
안녕하세요. 글 잘 읽었습니다. 다만, ξ(x)의 값이 정확히 이해가 되지 않습니다.
ξ(x) 가 구간 I 위에 존재하는 값이면 f(x)값과 동일한 것인가요?
또한, ξ0 은 ξ(x0)을 뜻하나요?

감사합니다.

**Reply by pisache** [reply]
수식 중 Three point formula에서 f'(x0)의 공식중 가운데 항은 4f(x0 + h) 이 맞지 않나요?
위 문의는 제가 따로 답을 찾았습니다. 감사합니다.
